packages/x-date-pickers/src/internals/hooks/useStaticPicker/useStaticPicker.types.ts

```diff
@@ -6,18 +6,11 @@ import {
 import { DateOrTimeView } from '../../models';
 import { BaseNextPickerProps } from '../../models/props/basePickerProps';
 import { UsePickerParams } from '../usePicker';
-import {
-  PickersSlotsComponent,
-  PickersSlotsComponentsProps,
-} from '../../components/wrappers/WrapperProps';
 
-export interface UseStaticPickerSlotsComponent
-  extends ExportedPickersViewLayoutSlotsComponent,
-    Pick<PickersSlotsComponent, 'PaperContent'> {}
+export interface UseStaticPickerSlotsComponent extends ExportedPickersViewLayoutSlotsComponent {}
```

Comment:
The `PaperContent` slot was never used on the new pickers
The goal of this slot was to be able to add custom content (like the shortcuts) in the layout. 
This will be handled by #6900 on the new pickers